USAGE.md

```diff
@@ -0,0 +1,83 @@
+# Getting Access to Data Lake
+Currently, Data Lake allows read/write access for all users from all AWS accounts (users home
+accounts) specified in `DATALAKE_USERS_AWS_ACCOUNTS_IDS` during deployment time
+(see [README](README.md#aws-infrastructure-deployment-cdk-stack)).
+
+Temporary access credentials for Data Lake can be requested by running following commands while
+authenticated using user's home AWS account credentials:
+```
+export DATALAKE_AWS_ACCOUNT_ID=<DATALAKE-AWS-ACCOUNT-ID>
+export DATALAKE_USER_ROLE_NAME=<DATALAKE-USER-ROLE-NAME>
+
+$ CREDENTIALS=$(aws sts assume-role \
+   --role-arn arn:aws:iam::${DATALAKE_AWS_ACCOUNT_ID}:role/${DATALAKE_USER_ROLE_NAME} \
+   --role-session-name datalake-user)
+
+$ export AWS_ACCESS_KEY_ID=$(echo $CREDENTIALS | jq -r ".Credentials[\"AccessKeyId\"]")
+$ export AWS_SECRET_ACCESS_KEY=$(echo $CREDENTIALS | jq -r ".Credentials[\"SecretAccessKey\"]")
+$ export AWS_SESSION_TOKEN=$(echo $CREDENTIALS | jq -r ".Credentials[\"SessionToken\"]")
+```
+
+# Data Lake Lambda Endpoints Usage
+
+## Endpoint Request Format
+```
+$ export DATALAKE_LAMBDA_FUNCTION_ENDPOINT_NAME=<DATALAKE-LAMBDA-FUNCTION-ENDPOINT-NAME>
```

Comment:
Do users need some instructions to retrieve the endpoint name? I'm not even completely sure I could get this command right.